packages/fxa-content-server/app/scripts/lib/constants.js

```diff
@@ -149,5 +149,8 @@ module.exports = {
   TWO_STEP_AUTHENTICATION_ACR: 'AAL2',
 
   STYLE_TRAILHEAD: 'trailhead',
+
+  // https://stripe.com/docs/error-codes#expired-card
+  CC_EXPIRED: 'expired_card',
```

Comment:
I'm not totally sure this will be the error string, although Les pointed me at [this line](https://github.com/mozilla/subhub/blob/master/subhub/api/payments.py#L216) in subhub, which suggests the stripe error codes are indeed passed through. We should confirm this with the subhub team. I filed https://github.com/mozilla/subhub/issues/148 to ask for some documentation.